packages/mobile/src/exchange/ExchangeConfirmationCard.test.tsx

```diff
@@ -24,7 +24,19 @@ it('renders correctly with no exchange rate', () => {
   expect(tree).toMatchSnapshot()
 })
 
-it('renders correctly with giant numbers', () => {
+it('renders correctly with larger numbers', () => {
+  const tree = renderer.create(
+    <Provider store={store}>
+      <ExchangeConfirmationCard
+        makerAmount={{ value: '240000.00', currencyCode: 'cUSD', localAmount }}
+        takerAmount={{ value: '180000000', currencyCode: 'cGLD', localAmount }}
+      />
+    </Provider>
+  )
+  expect(tree).toMatchSnapshot()
```

Comment:
Only checking with a snapshot doesn't tell much as we usually don't pay too much attention to them.
Would be better to also check the amount we expect is displayed.

Something like this:
```tsx
expect(getElementText(tree.getByTestId('SomeTestId'))).toEqual('$240,000')
```